docs/user_manual/retrohunt.md

```diff
@@ -0,0 +1,43 @@
+
+![Retrohunt](./images/retrohunt1.png)
+
+## Creating a new Retrohunt search
+
+To create a new search job, click the green "plus" button on the top right of the main interface to open the Create Retrohunt Search page. Enter the following details:
+
+- **Search information**: Provide a description for the search, select which indices to search on (search space) and enter the expiry date (days to live).
+- **Maximum file classification**: This property defines the scope of the included files in the search, but note that even with a higher classification level, users with a lesser classification will not be able to see all the resulting hits after the search has completed.
+- **YARA rule**: When writing a YARA rule, starting to type `rule` will show a snippet that will create a basic rule template. Write a valid rule otherwise submitting the request to create a search will fail.
+
+When you're done, click the "ADD RETROHUNT JOB" and confirm the job to create it. If no errors have occurred, a green snack bar should appear from the bottom and the interface should redirect to its Retrohunt Detail page. That page will show a progress bar denoting the status of the search. After the job has completed, it will show the resulting hits of the search.
+
+![Retrohunt](./images/retrohunt2.png)
+
+## Viewing the hits of a Retrohunt search
+
+The Retrohunt Detail page displays information in a tabular format.
+
+- **Details**: This tab shows the search information and displays the resulting hits of the search. You may reduce the scope of the hits by entering a search query or by clicking on a column in the distribution graphs to add a filter value. Clicking a on table row will open the detail page of that file.
+- **YARA Rule**: The entered YARA rule is in this tab to use the full size of the page making it easier to analyze.
+- **Errors**: This tab is only accessible to administrators as it shows the warnings and errors that occurred during the Retrohunt search process, which may be used for debugging purposes.
+
+![Retrohunt](./images/retrohunt3.png)
+
+## Repeating a Retrohunt search
+
+The Retrohunt feature offers the ability to repeat completed jobs using the same YARA rule. To repeat a search, click on the "Repeat this Retrohunt search" button on the top right side of the Retrohunt Detail page to open the "Repeat Retrohunt Search" dialog box. You may change the `maximum file classification` and the `days to live`, but note that selecting a lower value will not update that property even though the interface doesn't prevent you from selecting them.
```

Comment:
"selecting a lower value will not"

What value are we talking about here? Both the maximum file classification and the days to live?

Reply:
Either of them, yes.